ush/forecast_postdet.sh

```diff
@@ -439,11 +440,11 @@ EOF
   fi
 
   # Stochastic Physics Options
-  if [[ ${SET_STP_SEED:-"YES"} = "YES" ]]; then
+  if [[ ${DO_SPPT:-"NO"} = "YES" ]]; then
```

Comment:
```suggestion
 if [[ ${SET_STP_SEED:-"YES"} = "YES" ]]; then
 ISEED_SKEB=$((current_cycle*10000 + {MEMBER#0}*100 + 1))
 ISEED_SHUM=$((current_cycle*10000 + {MEMBER#0}*100 + 2))
 ISEED_SPPT=$((current_cycle*10000 + ${MEMBER#0}*100 + 3)),$((current_cycle*10000 + ${MEMBER#0}*100 + 4)),$((current_cycle*10000 + ${MEMBER#0}*100 + 5)),$((current_cycle*10000 + ${MEMBER#0}*100 + 6)),$((current_cycle*10000 + ${MEMBER#0}*100 + 7))
ISEED_CA=$(( (current_cycle*10000 + ${MEMBER#0}*100 + 4) % 2147483647 ))
ISEED_LNDP=$(( (current_cycle*10000 + ${MEMBER#0}*100 + 5) % 2147483647 ))
else
ISEED=${ISEED:-0}
fi
```

Reply:
Please indent correctly to make the code easy to read.

Reply:
SET_STP_SEED is not defined anywhere else in the workflow. This could be default to NO, and set YES in config.efcst? or keep as is to not introduce another variable

Reply:
We better keep it as is, set YES in efcst for default. It does not make sense we set all the seeds when DO_SPPT = YES.

Reply:
`ISEED_SKEB` should be created if `DO_SKEB` is `YES`.  Similarly for the `SHUM` and `SPPT` that have their respective `DO_` counterparts.

Reply:
The easiest way is to set it all without any condition, these seeds won't bother. 
If we add condition, we need to add condition for each of these stoch schemes.

Reply:
please see update and comment/suggest edits